osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -28,6 +28,11 @@ public class BeatmapDatabase : Database
         // ReSharper disable once NotAccessedField.Local (we should keep a reference to this so it is not finalised)
         private BeatmapIPCChannel ipc;
 
+        /// <summary>
+        /// A default representation of a WorkingBeatmap to use when no beatmap is available.
+        /// </summary>
+        public WorkingBeatmap DefaultBeatmap { get; set; }

```

Comment:
well it's kind of just a place to store the value. the BeatmapDatabase class can still work with it set to null (check where it's used internally) or any other default.